### PR TITLE
TST: add Appveyor CI (Windows, MSVC build)

### PR DESCRIPTION
**Pull request by rgommers**
For now 32-bit Python 2.7 only. Appveyor runtime is a bit limited and things don't go in parallel. This took 12 mins. Will optimize later (grabbing a Cython wheel would help a lot).

### CODE REVIEW

**Comment by rgommers** [comment]
Note: this built successfully once already: https://ci.appveyor.com/project/rgommers/pywt/build/1.0.8
Only took me 8 tries:)

Hat tip to @ogrisel.

**Comment by grlee77** [comment]
looks good to me.
Are you running only 32-bit because this is just to test against MSVC and the 64-bit case would be using the same compiler?

**Comment by rgommers** [comment]
I was just about to try enabling more builds, but I suspect that it will time out. This one build took 12 min, and the limit is 12 min. If I can get it to use the Cython wheel though instead of building Cython from source, then it'll work.

**Comment by rgommers** [comment]
giving it a try ....

**Comment by grlee77** [comment]
I wasn't aware of the 12 minute limit.  I tried looking on the appveyor site and found conflicting info.  The following page claims there is a 30 minute limit for free plans:
http://www.appveyor.com/docs/build-configuration#time-limitations
perhaps they have changed the limit?  I guess we will see!

**Comment by rgommers** [comment]
Sorry, typo - I meant 30 min. But I suspect it will still fail, because I'm now trying to do 4x 12 min.

**Comment by rgommers** [comment]
Success!

Still not sure why it's ignoring the Cython wheels though. It does take the numpy wheels which have an identical naming scheme.

**Comment by grlee77** [comment]
Maybe it is just that each individual case has to come in under 30 minutes.  If that is the case then we had plenty of time to spare.  The longest was 64-bit python 3 at 13 min, 26 sec.  The sum of the four was just over 40 minutes.

**Comment by rgommers** [comment]
Indeed, that must be it. So let's call this PR good and leave the Cython wheel thing for another time.

File: util/appveyor/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Fetch a numpy wheel from the sklearn rackspace wheelhouse.
+# That wheel was generated by @ogrisel by calling `wheel convert` on
+# the binaries from http://www.lfd.uci.edu/~gohlke/pythonlibs/
+# This is a temporary solution. As soon as numpy provides an official
+# wheel for windows we ca delete this --find-links line.
+--find-links http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/index.html
```
Comment (grlee77) [inline util/appveyor/requirements.txt]: I don't see a Cython wheel at the provided link (just some numpy and scipy ones):
http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/index.html
perhaps we need a separate link to a location providing that?  I don't have any prior experience for wheels or appveyor so I am not sure.

**Comment by grlee77** [comment]
fine with me.  Should I merge it as is or do you want to try to updating the Cython wheel link as mentioned above?

**Comment by rgommers** [comment]
Ah, it's the ``index.html`` that is the issue. The provided link in ``appveyor.yml`` (without index.html) is what I've been looking at: http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/

**Comment by rgommers** [comment]
I'll update requirements.txt now

**Comment by grlee77** [comment]
There appears to be a link at the following location if you don't already have it:
https://github.com/cython/cython-wheels

**Comment by rgommers** [comment]
Those are OS X and Linux wheels only it looks like - the commit I just pushed should fix things.

**Comment by rgommers** [comment]
Allright, this works fine and I'd like to get the wheels for v0.3.0 built automatically, so in it goes.

**Pull request merged**